Makefile

```diff
@@ -179,3 +179,13 @@ bdt:
 	bdt
 
 .PHONY: api-shell debug-api debug-worker docker-bootstrap docker-build lint load-fixtures load-fixtures-ssas load-synthetic-cclf-data load-synthetic-suppression-data package performance-test postman release smoke-test test unit-test worker-shell bdt unit-test-db unit-test-db-snapshot
+
+documentation:
+	# Rebuilds the swagger documentation when the local is currently running.
+	docker-compose up --build documentation
+	docker-compose up -d --build openapi
```

Comment:
I will look at refactoring this.  As it stands this is really made for local development.  We have to run this with in `detached` mode since it's just replacing the running containers in place.  Also we don't need the `--exit-code-from` here because it would just fail the build and show the error on screen.  I do like the idea from your other comment of keeping the documentation modular though.  Let me take a stab at refactoring this.

Reply:
I had the incorrect understanding of how `--exit-code-from` flag operates.  I used the same commands from the `docker-bootstrap` target to update the `documentation` target and then included removed the commands in place of the `documentation` target.